typography.scss

```diff
@@ -1,193 +1,141 @@
-$vui-help-fontWeight: normal !default;
-$vui-help-lineHeight: 20px !default;
-$vui-help-topMargin: 0 !default;
-$vui-help-rightMargin: 0 !default;
-$vui-help-bottomMargin: 20px !default;
-$vui-help-leftMargin: 0 !default;
-
-@mixin vui-typography(
-		$fontColor: $vui-fontColor,
-		$fontFamily: $vui-fontFamily,
-		$fontSize: $vui-fontSize,
-		$lineHeight: $vui-lineHeight
-	) {
-
-	color: $fontColor;
-	font-family: $fontFamily;
-	font-size: $fontSize;
-	font-weight: normal;
-	line-height: $lineHeight;
 
-}
-
-@mixin vui-typography-emphasis(
-		$fontColor: $vui-emphasis-fontColor,
-		$fontStyle: $vui-emphasis-fontStyle,
-		$topMargin: $vui-emphasis-topMargin,
-		$rightMargin: $vui-emphasis-rightMargin,
-		$bottomMargin: $vui-emphasis-bottomMargin,
-		$leftMargin: $vui-emphasis-leftMargin,
-		$baseFontSize: $vui-fontSize
-	) {
-
-	color: $fontColor;
-	font-style: $fontStyle;
-	margin: pxToEm($topMargin, $baseFontSize)
-			pxToEm($rightMargin, $baseFontSize)
-			pxToEm($bottomMargin, $baseFontSize)
-			pxToEm($leftMargin, $baseFontSize);
+@import 'bower_components/vui-colors/colors';
+@import 'px-to-rem';
+@import 'fonts';
+
+$vui-base-fontSize: 19px;
+
+$vui-fontColor: $vui-color-ferrite !default;
+$vui-fontFamily: 'Lato', 'Lucida Sans Unicode', 'Lucida Grande', sans-serif !default;
+$vui-fontSize: px-to-rem(19px, $vui-base-fontSize) !default;
+$vui-fontWeight: 300 !default;
+$vui-lineHeight: 1.5rem !default;
+$vui-letterSpacing: px-to-rem(0.3px, $vui-base-fontSize) !default;
+
+$vui-heading1-fontColor: $vui-fontColor !default;
+$vui-heading1-fontSize: px-to-rem(38px, $vui-base-fontSize) !default;
+$vui-heading1-fontWeight: $vui-fontWeight !default;
+$vui-heading1-lineHeight: 2rem !default;
+$vui-heading1-letterSpacing: px-to-rem(-0.5px, $vui-base-fontSize) !default;
+
+$vui-heading2-fontColor: $vui-fontColor !default;
+$vui-heading2-fontSize: px-to-rem(28px, $vui-base-fontSize) !default;
+$vui-heading2-fontWeight: $vui-fontWeight !default;
+$vui-heading2-lineHeight: 2rem !default;
+$vui-heading2-letterSpacing: px-to-rem(-0.3px, $vui-base-fontSize) !default;
+
+$vui-heading3-fontColor: $vui-fontColor !default;
+$vui-heading3-fontSize: px-to-rem(19px, $vui-base-fontSize) !default;
+$vui-heading3-fontWeight: 400 !default;
+$vui-heading3-lineHeight: 2rem !default;
+$vui-heading3-letterSpacing: px-to-rem(0.5px, $vui-base-fontSize) !default;
+
+$vui-small-text-fontColor: $vui-fontColor !default;
+$vui-small-text-fontSize: px-to-rem(14px, $vui-base-fontSize) !default;
+$vui-small-text-fontWeight: 400 !default;
+$vui-small-text-lineHeight: 1rem !default;
+$vui-small-text-letterSpacing: px-to-rem(0.2px, $vui-base-fontSize) !default;
+
+$vui-small-strong-text-fontColor: $vui-fontColor !default;
+$vui-small-strong-text-fontSize: px-to-rem(14px, $vui-base-fontSize) !default;
+$vui-small-strong-text-fontWeight: 700 !default;
+$vui-small-strong-text-lineHeight: 1rem !default;
+$vui-small-strong-text-letterSpacing: px-to-rem(0.4px, $vui-base-fontSize) !default;
```

Comment:
Agreed. Keeping them in when they're not needed (yet) increases your "API footprint" -- i.e. makes it harder to change/remove/refactor later because someone _might_ be using one of those variables.